packages/terra-functional-testing/lib/test-runner/test-runner.js

```diff
@@ -0,0 +1,48 @@
+const fs = require('fs');
+const path = require('path');
+const Launcher = require('@wdio/cli').default;
+
+class TestRunner {
+  /**
+   * Resolves the configuration file path.
+   * If no path is provided the current working directory will be searched. Falls back to the default configuration file.
+   * @param {string} configPath - The path to the test runner configuration file.
+   * @returns {string} A resolved path to the test runner configuration file.
+   */
+  static configPath(configPath) {
+    if (configPath) {
+      return path.resolve(configPath);
+    }
+
+    const relativeConfigPath = path.resolve(process.cwd(), 'wdio.conf.js');
+
+    if (fs.existsSync(relativeConfigPath)) {
+      return relativeConfigPath;
+    }
+
+    return path.resolve(__dirname, './wdio.conf.js');
+  }
+
+  /**
+   * Starts the test runner.
+   * @param {Object} options - The test run options.
+   * @returns {Promise} A promise that resolves with the test run exit code.
+   */
+  static async run(options) {
+    try {
+      const { config } = options;
+
+      const configPath = TestRunner.configPath(config);
+      const testRunner = new Launcher(configPath);
+
+      const exitCode = await testRunner.run();
+
+      process.exit(exitCode);
+    } catch (error) {
+      console.error('Launcher failed to start the test.\n', error);
```

Comment:
can we namespace with terra-functional-testing or the script name we decide on?

Reply:
I'll hard code a namespace for now. Long term we might want to handle these messages differently. Our Logger or one of the few debugging libraries available might be a good fit

Reply:
Namespaced the console message:
https://github.com/cerner/terra-toolkit/pull/406/commits/323a62a599edd71f5c21abcd5ba004bf9b8370eb